### PR TITLE
[FIX] Make Prettier work in wdio directory

### LINKED ISSUE #5518: Configure prettier on wdio (e2e tests folder)
**Description**
 Prettier is not working inside wdio directory.

**Acceptance Criteria**
* Prettier be working on wdio dir
* Feature file be covered
* E2E tests still running as they were running before the changes

**Good to have**
* Eslint error that came from removing wdio from ignore file be solved

### PR DESCRIPTION
**Pull request by NicolasMassart**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

_1. What is the reason for the change?_

Prettier formating is not running in `./wdio` directory because it's ignored.
Once removed from ignore file, it then doesn't formal all files as it has not formater for `feature` files.

_2. What is the improvement/solution?_

**Prettier related changes**

- remove wdio from the Prettier exclude file
- add Gherkin Prettier pluggin for .feature files
- update `yarn format` script to take .feature files into account
- first pass of js and feature files formating in wdio dir using `yarn format`

**Eslint side effect related changes**

Removing the wdio dir from the prettier ignore file had a side effect (the prettier ignore file was also used as ignore file for eslint) and raised eslint issues.

I decided to deal with eslint issues in another PR to prevent mixing topics and have this first #5518 issue fixed first.

The changes for this are the following:
- extract eslint config to deal with it in another PR
- create an eslint ignore file and use it as default instead of prettier ignore with eslint (on both scripts, lint and fix)
- add the wdio dir back in .eslintignore to deal with it in another eslint only PR
- remove useless ignore lines as already ignored by the android line

**Issue**

fixes #5518 

**Checklist**

* [x] There is a related GitHub issue
* [x] Tests are included if applicable
* [x] Any added code is fully documented

### CODE REVIEW

**Comment by NicolasMassart** [comment]
I have read the CLA Document and I hereby sign the CLA

File: wdio/features/AddressFlow.feature [hunk 1]
```diff
@@ -26,18 +27,22 @@ Feature: Adding addresses to contacts via address book modal and the contacts sc
 
   Scenario: A user adds an address to their contacts from the send flow and confirms it is visible on the contacts view
     When On the Main Wallet view I tap "Send"
+
```
Comment (Andepande) [inline wdio/features/AddressFlow.feature:30]: Were these spaces added by a cucumber formatter

File: .prettierignore [hunk 2]
```diff
@@ -1,4 +1,3 @@
-/android/app/src/main/assets/InpageBridgeWeb3.js
```
Comment (tommasini) [inline .prettierignore:1]: Why this line was removed?

Comment (NicolasMassart) [reply .prettierignore:1]: Because this ignore is already covered by the "android" line so it's not taken into account and useless

**Review by Andepande — APPROVED** [review]
Lgtm

**Pull request merged**